### DISCUSSION TITLE
Great Job

### DISCUSSION THREAD

**Discussion by PopeNobody** [Camelcade/Perl5-IDEA#2633] [General]
I've been playing with the IDEA plugin (I'm not sure what is in your other repos) and I'm loving it so far!

They should use it as the basis for a full fledged perl IDE.

**Comment by rwp0** [comment]
Yes, and unfortunately many Perlers are unaware thereof still thinking Vim and Emacs are the only worthy editors to write Perl code with.

**Comment by hurricup** [comment]
@rwp0  pretty often IDE/Editor selection is about love, not efficiency. It is irrational. 
@PopeNobody who are "they"? Nobody will develop commercial IDE for perl these days, too low demand.

**Reply by rwp0** [reply]
Hmm.. but then again we choose Perl because we love it, not because it's the most efficient programming/scripting language

**Reply by hurricup** [reply]
exactly my point :)
